src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../colors";
```

Comment:
We're also trying to implement some new typography globals using the `_typography` file. I'd recommend adding here:
```(css)
@import "../../typography";
```
Then, you can remove the styles below for `p`, `h1`, and `h2`.